packages/public/common-ui-web/src/components/Slider/useSliderStyle.ts

```diff
@@ -0,0 +1,135 @@
+import { CSSProperties, useMemo } from 'react';
+import { ColorProp, InteractiveStatus } from '@monkvision/types';
+import { useMonkTheme } from '@monkvision/common';
+import { styles } from './Slider.styles';
+
+/**
+ * Props that the Slider component can accept.
+ */
+export interface SliderProps {
+  /**
+   * The minimum value of the slider.
+   * @default 0
+   */
+  min?: number;
+
+  /**
+   * The maximum value of the slider.
+   * @default 100
+   */
+  max?: number;
+
+  /**
+   * The current value of the slider.
+   * @default (max - min) / 2
+   */
+  value?: number;
+
+  /**
+   * The primary color used for the background of the slider.
+   * @default 'secondary-xlight'
+   */
+  primaryColor?: ColorProp;
+
+  /**
+   * The secondary color used for the 2nd slider bar, growing depending on the value / thumb position.
+   * @default 'primary'
+   */
+  secondaryColor?: ColorProp;
+
+  /**
+   * The tertiary color used for the thumb/knob border .
+   * @default 'primary'
+   */
+  tertiaryColor?: ColorProp;
+
+  /**
+   * Determines if the slider is disabled.
+   * @default false
+   */
+  disabled?: boolean;
+
+  /**
+   * The increment value for the slider.
+   * @default 1
+   */
+  step?: number;
+
+  /**
+   * Callback function invoked when the slider value changes.
+   * @param value - The new value of the slider.
+   */
+  onChange?: (value: number) => void;
+
+  /**
+   * Optional styling
```

Comment:
Specify in this doc that the `width` property is used to set the width of the slider but the height does nothing